fuzzers/fuzzbench_text/Makefile.toml

```diff
@@ -0,0 +1,100 @@
+[env]
+FUZZER_NAME="fuzzer"
+PROJECT_DIR = { script = ["pwd"] }
+
+[tasks.unsupported]
+script_runner="@shell"
+script='''
+echo "Cargo-make not integrated yet on this"
+'''
+
+# Compilers
+[tasks.cxx]
+linux_alias = "cxx_unix"
+mac_alias = "cxx_unix"
+windows_alias = "unsupported"
+
+[tasks.cxx_unix]
+command = "cargo"
+args = ["build" , "--release"]
+
+[tasks.cc]
+linux_alias = "cc_unix"
+mac_alias = "cc_unix"
+windows_alias = "unsupported"
+
+[tasks.cc_unix]
+command = "cargo"
+args = ["build" , "--release"]
+
+# fuzz.o File
+[tasks.object]
+linux_alias = "object_unix"
+mac_alias = "object_unix"
+windows_alias = "unsupported"
+
+[tasks.object_unix]
```

Comment:
In a traditional makefile, these would be named `fuzz.o` after the output filename, ist that uncommon here?

Reply:
I can't use '.' (dot) for the task name.
but maybe fuzz_o then?

Reply:
Ah makes sense